apps/web-giddh/src/app/ledger/ledger.component.ts

```
@@ -356,6 +357,11 @@ export class LedgerComponent implements OnInit, OnDestroy {
     public ledgerGridColumnsValue: number[] = [1, 2, 1]
     /** Hold callback broadcast event */
     public callBackBroadcast: any;
+    /** Holds Bank Integration Dialog Ref */
+    public bankIntegrationDialogRef: any;
+    public isDirectlyIntegrated: boolean = false;
+    /** Observable for post balance success response */
+    public ledgerBalanceSuccess$: Observable<boolean> = this.ledgerComponentStore.select(state => state.ledgerBalance);
```

Biome (1.9.4)

[error] Duplicate class member name "ledgerBalanceSuccess$"



(lint/suspicious/noDuplicateClassMembers)